lms/djangoapps/instructor/tests/test_enrollment.py

```diff
@@ -756,6 +756,12 @@ def test_platform_language_is_used_for_logged_in_user(self):
             self.assertIn("You have been", subject)
             self.assertIn("You have been", message)
 
+    def test_platform_language_is_used_for_new_user(self):
+        # simulate a new user
+        subject, message = self.get_subject_and_message(None)
+        self.assertIn("You have been invited in platform language", subject)
+        self.assertIn("You have been invited in platform language", message)
```

Comment:
Sorry for my not fully considered codes, the new test case will check the system language code after render the message.